pkg/skaffold/runner/v1/new.go

```diff
@@ -89,6 +92,9 @@ func NewForConfig(runCtx *runcontext.RunContext) (*SkaffoldRunner, error) {
 	}
 
 	depLister := func(ctx context.Context, artifact *latestV1.Artifact) ([]string, error) {
+		ctx, endTrace := instrumentation.StartTrace(ctx, "pkg/skaffold/runner", "GetAllDependencies")
```

Comment:
```suggestion
		ctx, endTrace := instrumentation.StartTrace(ctx, "pkg/skaffold/runner", "depLister")
```

It would be really useful to be able to include both the artifact name and type as attributes to we can see if there are particular artifacts that are bad.  Perhaps we could have an `instrumentation.Anonymize()` that can be toggled off (with an env var?).  We'll need to schedule a review (go/anon).

Reply:
Adding a stub `PII(a.ImageName)` fxn for now.  Made #5885 to track the future work here